src/components/Menus/Sidebar/SettingsPanel/ThemeToggle/ThemeToggle.test.js

```diff
@@ -38,6 +44,12 @@ describe("When default theme is light mode and cookie unset", () => {
     expect(cookies.cookies.theme).toBe("dark");
   });
 
+  test("Fires theme updated custom event when button clicked", async () => {
+    const button = screen.getByText("sidebar.settingsMenu.themeOptions.dark");
+    fireEvent.click(button);
+    expect(themeUpdatedHandler).toHaveBeenCalled();
```

Comment:
💭 we want to test that light/dark is distinguishable too right?

Reply:
I actually tried to do this and it wasn't having it :/ Maybe worth a huddle this afternoon to see if we can get these tests to be more specific

Reply:
Right, have refactored slightly and got this working 🎉